modules/nextflow/src/main/groovy/nextflow/util/NameGenerator.groovy

```diff
@@ -397,6 +400,9 @@ class NameGenerator {
             // Margaret Hamilton - Director of the Software Engineering Division of the MIT Instrumentation Laboratory, which developed on-board flight software for the Apollo space program. https://en.wikipedia.org/wiki/Margaret_Hamilton_(scientist)
             "hamilton",
 
+            // Victoria "Vi" Hart - American mathematician and YouTuber. Recreational mathemusician, well-known for creating mathematical videos and popularizing mathematics. Hart founded the virtual reality research group eleVR and has co-authored several research papers on computational geometry and the mathematics of paper folding. https://en.wikipedia.org/wiki/Vi_Hart
+            "hart",
```

Comment:
Not sure we should add someone who is so contemporary, that could open it up to many people and make it hard to define who should be included

Reply:
I did add Pesquet already

Reply:
😂